Implementation of max_time (#740)
Hi-PACE/hipace: docs/source/run/parameters.rst

    Maximum number of time steps. `0` means that the 0th time step will be calculated, which are the
    fields of the initial beams.

* ``hipace.max_time`` (`float`) optional (default `infinity`)
    Maximum physical time of the simulation. If ``t + dt > hipace.max_time``, the time step ``dt``
    will be reduced so that ``t + dt = max_time``, both for the adaptive and a

>     fixed time step.

Reviewer: FYI the limitation to 100 characters does not apply to rest files, where it really doesn't make sense (and is particularly cumbersome). 1 sentence per line is a good way of doing it.

>     Maximum physical time of the simulation. If ``t + dt > hipace.max_time``, the time step ``dt``
>     will be reduced so that ``t + dt = max_time``, both for the adaptive and a
>     fixed time step.

Reviewer: Minor rewording
```suggestion
    Maximum physical time of the simulation. The ``dt`` of the last time step may be reduced so that ``t + dt = max_time``, both for the adaptive and a fixed time step.
```
